media-gfx/gimp/gimp-2.10.18.ebuild

```diff
@@ -0,0 +1,231 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
why not 7?

Reply:
This is due to inherit of autotools + gnome2. Last is not ported to EAPI 7 while gnome2.eclass functions is still required - it mainly used to reconfigure actions + disable deprication warnings. Early discussed with @a17r.

Reply:
ignore me, I firgot about it :/